docs/gitbook/from-zero-to-hero-tutorial/3.-training.md

```diff
@@ -250,7 +250,7 @@ class Cumulative(BaseStrategy):
         self.dataset = {}  # cumulative dataset
 
     def adapt_train_dataset(self, **kwargs):
-        super().adapt_train_dataset(**kwargs)
+        super().after_train_dataset_adaptation(**kwargs)
```

Comment:
I don't understand this call here.. is this a bug?

Reply:
Yes, the fzth needs to be updated after this change is merged.